pulsebot/bugzilla.py

```diff
@@ -14,6 +14,7 @@ class Bugzilla(object):
     def __init__(self, server, api_key):
         self._server = server.rstrip('/')
         self._session = requests.Session()
+        self._session.headers.update({'User-Agent': 'glandium/pulsebot'})
```

Comment:
I'd rather not hardcode `glandium` in there. Could you also add the rationale in the commit message? Is this for filtering on the bugzilla end? Can't filtering be done via api key?